app/Repositories/Eloquents/RoleUserEloquentRepository.php

```diff
@@ -24,14 +24,15 @@ public function store($roles, $userId)
         }
     }
 
-    public function destroy($userId){
+    public function destroy($userId)
+    {
         $role_users = $this->model()
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/RoleUserEloquentRepository.php</br>Variable "role_users" is not in valid camel caps format